lib/host/inject.js

```diff
@@ -1,11 +1,13 @@
 var injector = require('connect-injector');
 var debug = require('debug')('testee:html-injector');
+var path = require('path');
+var mime = require('mime-types');
 
 // Returns a middleware that injects the SocketIO JavaScript
 // and a link to the clients side adapters into any HTML file
 module.exports = function(configuration) {
   return injector(function(req, res) {
-    var header = res.getHeader('content-type');
+    var header = res.getHeader('content-type') || mime.contentType(path.extname(req.url));
```

Comment:
I think [url.parse](https://nodejs.org/api/url.html) is the appropriate tool here:

```js
var url = require('url');
mime.contentType(path.extname(url.parse(req.url).pathname));
```